src/python/zquantum/core/utils.py

```diff
@@ -647,3 +647,23 @@ def load_from_specs(specs: Specs):
     if isinstance(specs, str):
         specs = json.loads(specs)
     return create_object(specs)  # type: ignore
+
+
+def get_ordered_list_of_bitstrings(num_qubits: int):
```

Comment:
Please add typing and then remove "Returns" from the docstring.
```suggestion
def get_ordered_list_of_bitstrings(num_qubits: int) -> List[str]:
```